src/main/asciidoc/js/index.adoc

```diff
@@ -394,6 +394,17 @@ connection.execute(sql, function (execute, execute_err) {
 
 ----
 
+=== Multiple ResultSet responses
+
+In some cases your query might return more than one result set, in this case and to preserve the compatibility when
+the returned result set object is converted to pure json, the next result sets are chained to the current result set
+under the property `next`. A simple walk of all result sets can be achieved like this:
+
+[source,js]
+----
+Code not translatable
```

Comment:
code cannot be translated to other langs, an issue should be opened in the code translator for this (and a contribution welcome)